test/tailwind_formatter_test.exs

```diff
@@ -422,7 +422,7 @@ defmodule TailwindFormatterTest do
     """
 
     expected = ~S"""
-    <div class={"#{if false, do: "class"} tomato text-sm uppercase odd:decoration-slate-50 sm:lowercase sm:hover:bg-unknown-500 sm:hover:bg-gray-500 sm:disabled:text-lg sm:disabled:text-2xl lg:sm:dark:group-hover:disabled:text-blue-500 " <> " rounded-lg px-3 py-3 " <> " rounded-lg px-3 py-5 " <> " text-sm " <> " tomato"}></div>
+    <div class={"#{if false, do: "class"} tomato text-sm uppercase odd:decoration-slate-50 sm:lowercase sm:hover:bg-unknown-500 sm:hover:bg-gray-500 sm:disabled:text-lg sm:disabled:text-2xl lg:sm:dark:group-hover:disabled:text-blue-500" <> "rounded-lg px-3 py-3 " <> "rounded-lg px-3 py-5 " <> "text-sm" <> "tomato"}></div>
```

Comment:
The downside is that users can't write:

```elixir
"rounded" <> "text-sm"
```

and expect to get:

```elixir
"rounded " <> " text-sm"
```

I don't think this is an issue though. Users who aren't using TailwindFormatter already have to use spaces when concatenating, while current users already have spaces inserted by TailwindFormatter in their concatenations.

Reply:
It's a good idea to forgo control over the outer spaces, as otherwise there are a lot of cases one has to think about and account for 🙂 

Not sure how opinionated TailwindFormatter will become, but this is a sensible change until we reach the point where controlling the inner/outer spaces is desired for those using the library